controller/src/beerocks/master/wfa_ca.cpp

```diff
@@ -166,6 +175,60 @@ bool wfa_ca::parse_params(const std::vector<std::string> &command_tokens,
             return false;
         }
 
+        if ((*std::next(it))[0] == '{') {
+            int len_command_str     = (*std::next(it)).length();
+            bool prev_space         = false;
+            bool prev_curly_bracket = false;
+            int count_open_brackets = 0;
+            std::string command_str = *std::next(it);
+            for (int i = 0; i < len_command_str; i++) {
+                auto c = command_str[i];
+                LOG(ERROR) << "char: " << c;
+                if (c == '{') {
+                    prev_curly_bracket = true;
+                    prev_space         = false;
+                    count_open_brackets++;
+                } else {
+
+                    if (count_open_brackets == 0) {
+                        err_string =
+                            "The following string is not valid (curly brackets test failed) : '" +
+                            *std::next(it);
+                        return false;
+                    }
+                    if (c == '}') {
+                        count_open_brackets--;
+                        if (count_open_brackets < 0) {
+
+                            err_string = "The following string is not valid (curly brackets test "
+                                         "failed) : '" +
+                                         *std::next(it);
+                            return false;
+                        }
+                        prev_space = false;
+                    } else if (c == ' ') {
+                        prev_space = true;
+
+                    } else {
+                        if (prev_space == true || prev_curly_bracket == true) {
```

Comment:
IMO the comparison to true/false of booleans is not necessary.
if (VAR == true) -> if (VAR)
if (VAR == false) -> if (!VAR)